b2sdk/raw_simulator.py

```diff
@@ -307,7 +307,9 @@ def __init__(self, file_sim, url, range_=None):
 
     def iter_content(self, chunk_size=1):
         start = 0
+        rnd = random.Random(self.url)
```

Comment:
Why seed with the url?  So it's reproducible?

Reply:
Yes - the idea is that if I run test case two times it should run exactly the same way... we can either hardcode some arbitrary value here or use something from test case context... like file download url - I like more the second option :) but actually this is for forcing nonsequential writes in parallel download... and thread context switching is not deterministic... so it actually probably doesn't matter so much - but it is my pattern to seed random number generator from test case context...